templates/footer.html

```diff
@@ -4,82 +4,7 @@
-            Commons Attribution-ShareAlike 4.0 license.
-            </p>
-          </div>
-        </div>
-      </div>
-      <div class='links col-md-8'>
-        <div class='container'>
-          <div class='row'>
-            <div class='area col-md-3'>
-              <div class='div-mar'>
-                <h5>Resources</h5>
-                <ul class='resources'>
-                  <li><a href='https://www.jenkins.io/events'>Events</a></li>
-                  <li><a href='https://www.jenkins.io/doc'>Documentation</a></li>
-                  <li><a href='https://www.jenkins.io/node'>Blog</a></li>
-                </ul>
-              </div>
-            </div>
-            <div class='area col-md-3'>
-              <div class='div-mar'>
-                <h5>Solutions</h5>
-                <ul>
-                  <li><a href='https://www.jenkins.io/solutions/android'>Android</a></li>
-                  <li><a href='https://www.jenkins.io/solutions/c'>C/C++</a></li>
-                  <li><a href='https://www.jenkins.io/solutions/docker'>Docker</a></li>
-                  <li><a href='https://www.jenkins.io/solutions/embedded'>Embedded</a></li>
-                  <li><a href='https://www.jenkins.io/solutions/github'>GitHub</a></li>
-                  <li><a href='https://www.jenkins.io/solutions/java'>Java</a></li>
-                  <li><a href='https://www.jenkins.io/solutions/pipeline'>Continuous Delivery</a></li>
-                  <li><a href='https://www.jenkins.io/solutions/python'>Python</a></li>
-                  <li><a href='https://www.jenkins.io/solutions/ruby'>Ruby</a></li>
-                </ul>
-              </div>
-            </div>
-            <div class='area col-md-3'>
-              <div class='div-mar'>
-                <h5>Project</h5>
-                <ul class='project'>
-                  <li><a href='https://issues.jenkins-ci.org'>Issue tracker</a></li>
-                  <li><a href='https://wiki.jenkins-ci.org'>Wiki</a></li>
-                  <li><a href='https://github.com/jenkinsci'>GitHub</a></li>
-                  <li><a href='https://ci.jenkins.io'>Jenkins on Jenkins</a></li>
-                </ul>
-              </div>
-            </div>
-            <div class='area col-md-3'>
-              <div class='div-mar'>
-                <h5>Community</h5>
-                <ul class='community'>
-                  <li><a href='https://groups.google.com/forum/#!forum/jenkinsci-users'>
-                  Users mailing list</a></li>
-                  <li><a href='https://groups.google.com/forum/#!forum/jenkinsci-dev'>Developers mailing list</a></li>
-                  <li><a href='https://twitter.com/jenkinsci'>Twitter</a></li>
-                  <li><a href='https://reddit.com/r/jenkinsci'>Reddit</a></li>
-                </ul>
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</footer>
+<jio-footer property="https://jenkins.io"/>
```

Comment:
should be www to prevent a redirect and to match www.jenkins.io already
```suggestion
<jio-footer property="https://www.jenkins.io"/>
```